recipe/build.sh

```diff
@@ -11,12 +11,19 @@ CFLAGS="$CFLAGS -O3 -I$PREFIX/include"
 # its default rules for compiling C files.
 export TARGET_ARCH=
 
-make -j$CPU_COUNT LDFLAGS="$LDFLAGS" CFLAGS="$CFLAGS"
+# Set `CC` on Windows where it is not set already.
+if [ -z "$CC" ]; then
+    export CC="gcc"
+fi
```

Comment:
Had to manually set `CC` for the MSYS2 toolchain on Windows. Raised issue ( https://github.com/conda-forge/toolchain-feedstock/issues/59 ) to address this.